config/zones/RO.yaml

```diff
@@ -401,6 +426,8 @@ sources:
     link: https://docs.google.com/spreadsheets/d/1w5DJ7sPen6axIHU8TCVcuzNCjlct4I6JAbhUlw-ZXu8/edit?usp=sharing
   IPCC 2014:
     link: https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf#page=7
+  ANRE:
+    link: https://anre.ro/puteri-instalate/
```

Comment:
This is not for capacities but rather carbon intensities.

Please remove it from here.